schema/site.schema.json

```
@@ -557,7 +557,14 @@
     "auth.enableUsernameChanges": {
       "description": "Enables users to change their username after account creation. Warning: setting this to be true has security implications if you have enabled (or will at any point in the future enable) repository permissions with an option that relies on username equivalency between Sourcegraph and an external service or authentication provider. Do NOT set this to true if you are using non-built-in authentication OR rely on username equivalency for repository permissions.",
       "type": "boolean",
-      "default": false
+      "default": false,
+      "group": "Authentication"
+    },
+    "auth.minPasswordRunes": {
```

I don't think non-Go users would know what `Runes` are. How about just using "password length"?

```suggestion
    "auth.minPasswordLength": {
```

The "description" can explain that it's Unicode code points that are being counted.